presto-native-execution/presto_cpp/main/types/PrestoToVeloxSplit.cpp

```diff
@@ -91,6 +101,34 @@ velox::exec::Split toVeloxSplit(
             serdeParameters),
         splitGroupId);
   }
+
+  if (auto icebergSplit =
+          std::dynamic_pointer_cast<const protocol::IcebergSplit>(
+              connectorSplit)) {
+    std::unordered_map<std::string, std::optional<std::string>> partitionKeys;
+    for (const auto& entry : icebergSplit->partitionKeys) {
```

Comment:
Does this not work?
`partitionKeys.emplace(
          entry.second.name,
          entry.second.value);`

Reply:
No, since the second argument requires an optional string it throws an error.